resources/istio/values.yaml

```diff
@@ -171,7 +171,7 @@ components:
 
 global:
   containerRegistry:
-    path: eu.gcr.io/kyma-project
+    path: europe-docker.pkg.dev/kyma-project/prod
```

Comment:
This is overwritten in the images, but why do we add `prod` here? If we remove the `containerRegistryPath` it would fail, since the directory already has `prod`defined, right? 🤔

Reply:
`/prod` is a mistake in here, we actually do not use this property `containerRegistry` at all in istio helm templates. let see if it fails without it.